### ISSUE TITLE
Dark Mode for Boarding Record page

### ISSUE THREAD

**Issue by o-fish-wildaid** [WildAid/o-fish-android#376]
This issue depends on #368 being completed. 

When Dark Mode is on - the Boarding Record page should look like this: https://mongodb.invisionapp.com/share/PCYZ6ZW7FR8#/screens/433591784_Boarding_Record

Color key for light mode: https://mongodb.invisionapp.com/share/PCYZ6ZW7FR8#/screens/433591768_Color_Key_-_Light_Mode
Color key for dark mode: https://mongodb.invisionapp.com/share/PCYZ6ZW7FR8#/screens/433591769_Color_Key_-_Dark_Mode
Component Color specs for light mode: https://mongodb.invisionapp.com/share/PCYZ6ZW7FR8#/screens/433591770_Component_Colors_-_Light_Mode
Component Color specs for dark mode: https://mongodb.invisionapp.com/share/PCYZ6ZW7FR8#/screens/433591771_Component_Colors_-_Dark_Mode

Both dark and light modes should follow the color key and specs above (it is possible that the current light mode is not), and colors should be stored in:
app/src/main/res/values-night/colors.xml 
app/src/main/res/values/colors.xml 

Please make sure to TEST your build on a device or emulator to verify everything has changed - you can get a login and realm-app-id with easy instructions at https://wildaid.github.io/contribute/sandbox.html

**Comment by cfsnsalazar** [comment]
I would like to start working on this issue. Can it be assigned to me?

**Comment by ippschi** [comment]
Since I wrote about my concerns regarding the colors and the future use of them I just created a pull request with the proposal of having a separate colors file for the "brand" colors. Basically all the colors defined by the designers in invisionapp. (This can also be exported if not mistaken).
The PR to fix the issue will follow shortly

**Comment by cfsnsalazar** [comment]
Ok. So, should I close my current PR then? Should I modify it?

**Comment by Sheeri** [comment]
Hi @cfsnsalazar - does your pr at https://github.com/WildAid/o-fish-android/pull/402 fix this issue?

**Comment by cfsnsalazar** [comment]
@Sheeri  I tried solving it on that PR. However, I have questions as I am still familiarizing myself with the repo. It seems that most of the work was done. Most of the work that I did was related to the Toolbar 

Also,  I did not touch SafetyColorTextView as it seems that this view touches on other fragments as well.

I left some questions in the PR. Let me know If I am missing something. I don't mind starting again if necessary
